packages/gatsby-link/src/__tests__/index.js

```diff
@@ -19,6 +19,12 @@ const getNavigateTo = () => {
 }
 
 describe(`<Link />`, () => {
+
+  it(`does not fail to initialize when __PREFIX_PATHS__ is not defined`, () => {
```

Comment:
Another way you could write this (I think!) is like this

expect(() => {
  const Link = require(`../`).default
  const link = new Link({})
}).not.toThrow();

So that you're actually writing a test and not relying on side effects or anything. Also probably want to drop the eslint comments from your commit!